pipelines/test-release.yml

```diff
@@ -0,0 +1,77 @@
+trigger:
+  branches:
+    exclude:
+    - '*'
+
+stages:
+- stage: Test
+  jobs:
+  - job: testextension 
+    steps:
+    - task: NLUTrain@0
+      inputs: 
+        service: luis
+        modelSettings: models/luismodel.json
+    - task: NLUTest@0
+      inputs:
+        service: luis
+        utterances: models/tests.json
+        publishTestResults: true
+    - task: NLUClean@0
+      inputs:
+        service: luis  
+
+- stage: Release
```

Comment:
See question below, not sure we need this stage in a CI pipeline if we'll have a gated release build that deploys from a validated published artifact.

Reply:
Will use manual approval as discussed